### DISCUSSION TITLE
Can i get info from a "selected" body ?

### DISCUSSION THREAD

**Discussion by Gorhk** [EDCD/EDDI#2241] [EDDI Standalone Q&A] [answered]
i there, i'm looking for adding some cool stuffs in EDDI, i've already succed to get the amount of "atmospheric && landable planets", on a system, retreive their name and atmosphere composition etc.

but i'd like to be able to get informations about the "selected" planet, by selected i mean, "targeted" in supercruise.. not in discovery scan or advanced planetary scanner..

 amount of infos will probably depend of the level of scan i've done at the moment, but is it possible to get the "BodyReport" of the "targeted" planet ?

**Comment by Tkael** [comment, marked as answer]
I don't believe that this is currently possible. Unlike with ships, there is no journal event for targeted bodies.
